### PR TITLE
Add logging to service fetch action

### LINKED ISSUE #2681: Add logging to Service Fetch Action
Service fetch logic action needs to log when fetches are performed.

Ensure that the full logic evaluation logs with context are visible in the submission admin > logs as well. Currently they show that the logic rule was triggered, but no additional information is visible. This should log the resulting extracted value at minimum. Also add python level logging

### PR DESCRIPTION
**Pull request by CharString**
Closes #2681

### CODE REVIEW

File: src/openforms/submissions/logic/actions.py [hunk 1]
```diff
@@ -225,16 +226,37 @@ class ServiceFetchAction(ActionOperation):
...
         # XXX perform_service_fetch is expressed in terms of FormVariables
         dummy_var = FormVariable(
             name=self.variable,
             service_fetch_configuration=ServiceFetchConfiguration.objects.get(
                 pk=self.fetch_config
             ),
         )
-        with log_errors({}, self.rule):  # TODO proper logging/error handling
-            return {dummy_var.name: perform_service_fetch(dummy_var, context)}
+        with log_errors({}, self.rule):  # TODO proper error handling
```
Comment (CharString) [inline src/openforms/submissions/logic/actions.py:241]: This context manager is a bit to concrete.
I'm not sure whether I should change it, or turn this instance into a try-except.

We might even do a `log()` call on exception and log the error to the history in the admin.

File: src/openforms/submissions/logic/actions.py [hunk 2]
```diff
@@ -225,16 +226,37 @@ class ServiceFetchAction(ActionOperation):
...
+            result = perform_service_fetch(dummy_var, context)
+            log(asdict(result))
```
Comment (CharString) [inline src/openforms/submissions/logic/actions.py:243]: This is the most general thing that gives useful debugging information. I'm not sure if extra markup is wanted in the History view in  the admin.

File: src/openforms/submissions/logic/actions.py (hunk 1 above)
Comment (sergei-maertens) [reply src/openforms/submissions/logic/actions.py:241]: the admin `log` call makes sense, but it should be generic, something along the lines of "Service fetch action crashed (contact technical support)."

It kind of depends if it's a crash caused by bad logic rules, or by bugs by developers. The logic log page is meant for form designers so they can test their forms and see that their logic rules are behaving as they expect.

File: src/openforms/submissions/logic/actions.py [hunk 3]
```diff
@@ -190,29 +194,26 @@ class VariableAction(ActionOperation):
...
-            action_logic_introspection = introspect_json_logic(
-                self.value, component_map, initial_data
-            )
-            value = action_logic_introspection.as_string()
-
         return {
             "key": self.variable,
             "type_display": LogicActionTypes.get_label(LogicActionTypes.variable),
-            "value": value,
+            "value": log_data["value"],
```
[the remark this answers is not in the document]
Comment (sergei-maertens) [reply src/openforms/submissions/logic/actions.py:216]: I'll spin up or check a 2.0.x instance to check how this was supposed to look so we can get that fixed in master then.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**